### PR TITLE
Move application configuration vars to platform_info header

### PR DESCRIPTION
**Pull request by bentheredonethat**
Make the apps easier to debug and configure by moving all of the relevant variables to one file.

This mimics ST openamp apps

### CODE REVIEW

File: apps/machine/zynq7/rsc_table.c [hunk 1]
```diff
@@ -29,8 +29,6 @@
 /* Resource table entries */
 #define NUM_VRINGS                  0x02
 #define VRING_ALIGN                 0x1000
```
Comment (TanmayShah-xilinx) [inline apps/machine/zynq7/rsc_table.c:31]: You can move VRING_ALINE and NUM_VRINGS as well.

File: apps/machine/zynqmp/platform_info.h [hunk 2]
```diff
@@ -51,6 +51,63 @@ struct remoteproc_priv {
 #define POLL_STOP 0x1U
 #endif /* RPMSG_NO_IPI */
 
+#define RPU_CPU_ID          0 /* RPU remote CPU Index. We only talk to
```
Comment (TanmayShah-xilinx) [inline apps/machine/zynqmp/platform_info.h:54]: where is this used? Can we use XPAR_CPU_ID ?

Comment (bentheredonethat) [reply apps/machine/zynqmp/platform_info.h:54]: XPAR_CPU_ID is not provided in linux.

**Comment by bentheredonethat** [comment]
@TanmayShah-xilinx I resolved the vring comment. The one about Linux due to the reason I mentioned above.

File: apps/machine/zynq7/rsc_table.c (hunk 1 above)
Comment (bentheredonethat) [reply apps/machine/zynq7/rsc_table.c]: done in the next round of commits

File: apps/machine/zynq7/platform_info.h [hunk 3]
```diff
@@ -11,11 +11,17 @@
...
 
 #if defined __cplusplus
 extern "C" {
 #endif
 
+#define RING_TX                     FW_RSC_U32_ADDR_ANY
+#define RING_RX                     FW_RSC_U32_ADDR_ANY
+#define NUM_VRINGS                  0x02
+#define VRING_ALIGN                 0x1000
+
```
Comment (arnopo) [inline apps/machine/zynq7/platform_info.h:24]: In apps/machine/zynqmp_r5/platform_info.h you also move the  VRING_SIZE, What about doing same here?

**Comment by bentheredonethat** [comment]
closing for now.

**Pull request closed without merging**